backend/stan-wasm-server/src/app/config.py

```diff
@@ -22,6 +23,15 @@ class StanWasmServerSettings(BaseSettings):
     tinystan: DirectoryPath = Field(
         validation_alias=AliasChoices("tinystan", "tinystan_dir")
     )
+    log_level: str = "INFO"
+
+    @field_validator("log_level")
+    @classmethod
+    def log_level_is_valid(cls, v: str) -> int:
+        v = v.upper()
+        if v not in ("DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"):
+            raise ValueError(f"Invalid log level: {v}")
```

Comment:
This is a little magic--validating the (string) envvar value, then looking up against the module for a (hopefully extant) numeric code corresponding to that value, then preserving that in the settings object (despite the field being annotated as `str`). Might want to leave a note for this :)

Reply:
I re-thought this, give it a fresh look?

Reply:
I think this is a lot clearer--nice.